src/core/Akka.TestKit/TestKitBase_ExpectAllWithPredicates.cs

```diff
@@ -48,7 +48,7 @@ public IReadOnlyCollection<object> ExpectMsgAllOfMatchingPredicates(TimeSpan max
     {
         return ExpectMsgAllOfMatchingPredicatesAsync(max, predicates, cancellationToken)
             .ToListAsync(cancellationToken)
-            .ConfigureAwait(false).GetAwaiter().GetResult();
+            .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.